### DISCUSSION TITLE
Update without internet access

### DISCUSSION THREAD

**Discussion by JamesEdmondsUKPN** [requarks/wiki#6752] [Help / Questions] [answered]
Hi,

Is there a way to update a Windows installation without internet access?

Many thanks
James

**Comment by NGPixel** [comment, marked as answer]
Follow the [upgrade instructions](https://docs.requarks.io/install/upgrade) for Windows, but do the step where it downloads the latest tarball from another computer. Then transfer that file to your computer without internet access.
